pkg/target/regolib/src.rego

```diff
@@ -320,10 +324,35 @@ matches_namespaces(match) {
 
 matches_namespaces(match) {
   has_field(match, "namespaces")
+  # JULIAN - I can't figure out why we care about namespace selectors here.  Is that because nobody uses namespaceSelectors alongside namespaces?
   not always_match_ns_selectors(match)
   get_ns_name[ns]
   nss := {n | n = match.namespaces[_]}
-  count({ns} - nss) == 0
+  # count({ns} - nss) == 0
+  # JULIAN - this makes way more sense to me
+  nss[ns]
+}
+
+# Prefix-based matching
+matches_namespaces(match) {
+  has_field(match, "namespaces")
+  not always_match_ns_selectors(match)
+  get_ns_name[ns]
+  wild_nss := wildcard_namespaces(match.namespaces)
+  prefix_glob_match(wild_nss, ns)
+}
+
+wildcard_namespaces(ns_array) = out {
+  out := [ nss | endswith(ns_array[i], "*")
+                         nss := ns_array[i]]
+}
+
+prefix_glob_match(match_nss, object_ns) = true {
```

Comment:
`= true` is implicit

Reply:
There are a bunch of these around.  I'll remove them.